manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://wingettest.z6.web.core.windows.net/NXT_Installer_2.1.0_Global_WIN.exe

```

Comment:
This doesn't look like an official URI from the publisher. The link I found was `https://education.lego.com/_/downloads/NXT_Installer_2.1.0_Global_WIN.zip`, which would be blocked until we get .zip support.

Reply:
Therefore, I tried repacking the app. Do you have a timeline on implementation of .zip support?